raft/raft_test.go

```diff
@@ -589,8 +589,10 @@ func TestHandleHeartbeat2AA(t *testing.T) {
 		m       pb.Message
 		wCommit uint64
 	}{
-		{pb.Message{From: 2, To: 1, MsgType: pb.MessageType_MsgHeartbeat, Term: 2, Commit: commit + 1}, commit + 1},
-		{pb.Message{From: 2, To: 1, MsgType: pb.MessageType_MsgHeartbeat, Term: 2, Commit: commit - 1}, commit}, // do not decrease commit
+		{pb.Message{From: 2, To: 1, MsgType: pb.MessageType_MsgHeartbeat, Term: 2, LogTerm: 3, Commit: commit + 1}, commit + 1},
```

Comment:
```suggestion
		{pb.Message{From: 2, To: 1, MsgType: pb.MessageType_MsgHeartbeat, Term: 3, LogTerm: 3, Commit: commit + 1}, commit + 1},
```
Leader's term should be greater or equal to `LogTerm`